portfolio/src/main/webapp/get-comments.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    
+  </head>
+  <body>
+     
+    <button id="why"  class = "intro" onclick = "getComments()"> GET</button>
+    <button id="whyw"  class = "intro" onclick = "deleteComments()"> DELETE</button>
```

Comment:
can you use more precice IDs here? maybe "get-comments-btn" "delete-comments-btn" etc?